tests/integration/standard/test_metadata.py

```diff
@@ -1207,7 +1207,7 @@ def test_export_keyspace_schema_udts(self):
         cluster.shutdown()
 
     @greaterthancass21
-    @pytest.mark.xfail(reason='Column name in CREATE INDEX is not quoted. It\'s a bug in driver or in Scylla')
+    @xfail_scylla(reason='Column name in CREATE INDEX is not quoted. It\'s a bug in driver or in Scylla')
```

Comment:
Please create and issue (in the driver repo) for that and reference it here.

Reply:
you already adding it in other PR
https://github.com/scylladb/python-driver/pull/363/files#diff-10e4dc66833987dff353dedf3a30e98f8050a7e36b8cc6744d08c0f2552cbe11R1211

it was a scylla issue, that was fixed in 5.2